frontend/serve/main.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/short-d/app/fw/env"
+	"github.com/short-d/app/fw/envconfig"
+	"github.com/short-d/app/fw/router"
+	"github.com/short-d/app/fw/service"
+	"github.com/short-d/short/frontend/serve/handle"
+	"github.com/short-d/short/frontend/serve/shortapi"
+	"github.com/short-d/short/frontend/serve/ssr"
+)
+
+func main() {
+	goDotEnv := env.NewGoDotEnv()
+	envConfig := envconfig.NewEnvConfig(goDotEnv)
+
+	config := struct {
+		GRPCHostName string `env:"GRPC_HOST_NAME" default:"localhost"`
+		GRPCPort     int    `env:"GRPC_PORT" default:"8081"`
+		HTTPPort     int    `env:"HTTP_PORT" default:"3000"`
+	}{}
```

Comment:
```suggestion
	goDotEnv := env.NewGoDotEnv()
	envConfig := envconfig.NewEnvConfig(goDotEnv)

	config := struct {
		ShortRPCAPIHost string `env:"SHORT_RPC_API_HOST" default:"localhost"`
		ShortRPCAPIPort     int    `env:"SHORT_RPC_API_PORT" default:"8081"`
		HTTPPort     int    `env:"HTTP_PORT" default:"3000"`
	}{}
```
Just to avoid confusion in case people thought the gRPC config is for the frontend server.